docs/gerrit.md

```diff
@@ -0,0 +1,115 @@
+# Using Jujutsu with Gerrit Code Review
+
+JJ and Gerrit share the same mental model, which makes Gerrit feel like a
+natural collaboration tool for JJ. JJ tracks a "change identity" across
+rewrites, and Gerrit’s `Change-Id` tracks the same logical change across patch
+sets. JJ and Gerrit's `Change-Id`s aren’t natively compatible yet, but they’re
+philosophically aligned. `jj gerrit upload` bridges the gap today by adding a
+Gerrit-style `Change-Id` while JJ keeps its own notion of change identity on the
+client. In practice, that means small, clean commits that evolve over
+time—exactly how Gerrit wants you to work.
+
+This guide assumes a basic understanding of Git, Gerrit, and Jujutsu.
+
+## Set up a Gerrit remote
+
+Jujutsu communicates with Gerrit by pushing commits to a Git remote. If you're
+starting from an existing Git repository with Gerrit remotes already configured,
+you can use `jj git init --colocate` to start using JJ in that repo. Otherwise,
+set up your Gerrit remote.
+
+```shell
+# Option 1: Start JJ in an existing Git repo with Gerrit remotes
+$ jj git init --colocate
+
+# Option 2: Add a Gerrit remote to a JJ repo
+$ jj git remote add gerrit ssh://gerrit.example.com:29418/your/project
```

Comment:
```suggestion
$ jj git remote add gerrit https://review.gerrithub.io/yourname/yourproject
```

If you use Git/HTTPS is going to be more similar to a real production use; also using GerritHub.io could help people experimenting it without having to setup Gerrit at all.